### ISSUE TITLE
Extracting a single value from a ``DateTimeNano`` does not return a scalar

### ISSUE THREAD

**Issue by jack-pappas** [rtosholdings/riptable#234]
When you have an ``rt.DateTimeNano`` array and you extract a single element (e.g. ``my_dtn[3]``), the expectation is that you'd get back an ``rt.DateTimeNanoScalar`` result which preserves the semantics of the data + the timezone. In riptable 1.1.3, this operation is returning a raw integer value instead.

I haven't checked the behavior for ``rt.Date``, ``rt.TimeSpan``, or ``rt.DateSpan``, but they all have equivalent scalar types and are expected to have the same behavior described above.

**Comment by RCMcNamara** [comment]
Looks like this is fixed in 1.3.3 (or at least I am not having this problem)

**Comment by OrestZborowski-SIG** [comment]
This does appear to be resolved:
```
import riptable as rt

for arr in [rt.DateTimeNano([1647603828000000000]), rt.Date([12345]), rt.DateSpan([1647603828])]:
    print(f"arr = {repr(arr)}, type = {type(arr)}")
    val = arr[0]
    print(f"val = {repr(val)}, type = {type(val)}")
```
emits
```
arr = DateTimeNano(['20220318 07:43:48.000000000'], to_tz='NYC'), type = <class 'riptable.rt_datetime.DateTimeNano'>
val = DateTimeNanoScalar('20220318 07:43:48.000000000'), type = <class 'riptable.rt_datetime.DateTimeNanoScalar'>
arr = Date(['2003-10-20']), type = <class 'riptable.rt_datetime.Date'>
val = 2003-10-20, type = <class 'riptable.rt_datetime.DateScalar'>
arr = DateSpan(['1647603828 days']), type = <class 'riptable.rt_datetime.DateSpan'>
val = 1647603828 days, type = <class 'riptable.rt_datetime.DateSpanScalar'>
```
